tests/unit/components/common/StatusList.spec.ts

```diff
@@ -29,4 +32,33 @@ describe('StatusList.vue', () => {
     })
     expect(wrapper.find('[data-test="select-status"]').exists()).toBeTruthy()
   })
+
+  it('removes Refund Rejected option', async () => {
+    // Mock the API call to return the list of routing slip statuses
+    const sandbox = sinon.createSandbox()
+    const get = sandbox.stub(CodesService, 'getCodes')
+    get.returns(new Promise(resolve => resolve({ data: { codes: routingSlipStatusListMock } })))
+
+    // Render the StatusList component
+    const wrapper = mount(StatusList, {
+      localVue,
+      vuetify,
+      propsData: {
+        value: 'ACTIVE'
+      }
+    })
+
+    // Wait for the component to finish rendering
+    await wrapper.vm.$nextTick()
+
+    // Get all of the descriptions from the rendered status list
+    const descriptions = wrapper.vm.routingSlipStatusList.map(status => status.description)
```

Comment:
using `wrapper.vm.$data.routingSlipStatusList` instead should fix the build errors